src/client/src/apps/StyleguideRoute/styled/index.tsx

```diff
@@ -1,12 +1,12 @@
 // For inclusion in rt-styleguide package
 export { Block } from './Block'
-export type { BlockProps } from './Block'
+export * from './Block'
 export { Paragraph } from './Paragraph'
-export type { ParagraphProps } from './Paragraph'
+export * from './Paragraph'
 export { Text, mapTextProps } from './Text'
-export type { TextProps } from './Text'
+export * from './Text'
 export { mapMarginPaddingProps } from './mapMarginPaddingProps'
-export type { MarginPaddingProps, MarginProps, PaddingProps } from './mapMarginPaddingProps'
+export * from './mapMarginPaddingProps'
```

Comment:
Why are we changing these to star exports @rafgarciaa ? Isn't that frowned upon?

Reply:
I'm getting TS errors for some reason